File: Web_search_job/DTO/Employer/CommentDTO.cs
using Web_search_job.DTO.User;

namespace Web_search_job.DTO.Employer
{
    public class CommentDTO
    {
        public int? Id { get; set; }
        public long? CommentStars { get; set; }
        public string CommentTitle { get; set; }
        public string CommentText { get; set; }
        public DateTime CommentsToEmployerCreatedAt { get; set; }

        public CommentUserDTO User { get; set; }
    }
}

File: Web_search_job/ClientApp/src/app/models/backend/dtos/employers/Comment.dto.ts
import {CommentUserDTO} from "../users/comment-user.dto";

export interface CommentDTO {
  id: number;
  user: CommentUserDTO | null;
  commentStars: number;
  commentTitle: string;
  commentText: string;
  commentsToEmployerCreatedAt: Date;
}
